src/particle/particle/particle.py

```diff
@@ -534,9 +534,10 @@ def to_dict(cls, *args, **kwargs):
         """
         query_as_list = cls.to_list(*args, **kwargs)
 
-        return dict(
-            zip(query_as_list[0], zip(*query_as_list[1:]))
-        )  # dict(zip(keys, values))
```

Comment:
If you have to describe what you mean in a code-like comment - just write it that way in the code! :)

Reply:
Fair enough.